1_Installation_Files (v1.5.5)/YOUR_ADMIN/includes/init_includes/init_image_handler.php

```diff
@@ -261,7 +261,7 @@
             zen_register_admin_page('toolsImageHandlerUninstall', 'BOX_TOOLS_IMAGE_HANDLER_UNINSTALL', 'FILENAME_IMAGE_HANDLER_UNINSTALL', '', 'tools', 'N', 99);
         }
         
-        if (version_compare(IH_VERSION, '5.0.0', '<')) {
+        if (version_compare(IH_VERSION, '5.0.0', '<') || !defined('IH_CACHE_NAMING')) {
```

Comment:
Why did you need to add that additional clause?

Reply:
Trying to save an insert and an update.

Reply:
Should i have left the original insert?

Reply:
The initialization only 'kicks in' if the version of IH is different from that previously recorded in the database, so there's no need to 'save'.

You just need to remove that added clause.